.travis.yml

```diff
@@ -57,7 +68,7 @@ script:
   # -batch is necessary to make sure sbt doesn't stall waiting for user input in case of certain failures
   - echo '-batch' >> .sbtopts
   - sbt scalafmtCheck
-  - sbt compile test
+  - sbt compile
```

Comment:
We've got no actual tests, right? :trollface:

Reply:
![image](https://user-images.githubusercontent.com/15652496/89872779-bf1cf480-dbb9-11ea-97e4-ac67a273cea1.png)